pkg/e2e/openshift/hypershift/verifyInstall.go

```diff
@@ -0,0 +1,85 @@
+// Package openshift runs the OpenShift extended test suite.
+package hypershift
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	viper "github.com/openshift/osde2e/pkg/common/concurrentviper"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+
+	"github.com/openshift/osde2e/pkg/common/alert"
+	"github.com/openshift/osde2e/pkg/common/aws"
+	"github.com/openshift/osde2e/pkg/common/cluster"
+	"github.com/openshift/osde2e/pkg/common/config"
+	"github.com/openshift/osde2e/pkg/common/util"
+)
+
+var (
+	hypershiftInstallVerify = "[Suite: hypershift]"
+	workerNodes             []string
+)
+
+func init() {
+	alert.RegisterGinkgoAlert(hypershiftInstallVerify, "SD-CICD", "Diego Santamaria", "sd-cicd-alerts", "sd-cicd@redhat.com", 4)
+}
+
+// Checks the installation of the hypershift cluster
+var _ = ginkgo.Describe(hypershiftInstallVerify, func() {
+	ginkgo.Context("Verify install using oc", func() {
+		util.GinkgoIt("Worker nodes are ready", func(ctx context.Context) {
+			Expect(checkWorkerNodesInCluster()).ToNot(BeEmpty())
+		}, float64(viper.GetFloat64(config.Tests.PollingTimeout)))
+	})
+
+	ginkgo.Context("Verify install using AWS", func() {
+		util.GinkgoIt("Worker nodes are present in CCS account", func(ctx context.Context) {
+			Expect(checkWorkerNodesInAWS()).To(BeTrue())
+		}, float64(viper.GetFloat64(config.Tests.PollingTimeout)))
+	})
+})
+
+// checkWorkerNodesInCluster returns a list of nodes in the cluster
+func checkWorkerNodesInCluster() ([]string, error) {
```

Comment:
```suggestion
func checkWorkerNodesInCluster(ctx context.Context) ([]string, error) {
```

can pass through the ginkgo context to use it below